lib/array.js

```diff
@@ -206,6 +206,46 @@ helpers.forEach = function(array, options) {
 };
 
 /**
+ * Block helper that groups array elements by given `value`. 
+ *
+ * Given the array `['a', 'b', 'c', 'd','e','f','g','h']`:
+ *
+ * ```handlebars
+ * {{#groupEach 4 array}}
+ *   {{#each this}}
+ *     {{.}}
+ *   {{each}}
+ *   <br>
+ * {{/groupEach}}
+ * ```
+ *  //=> 'a','b','c','d'<br>
+ *  //=> 'e','f','g','h'<br>
+ *
+ * @name .inArray
+ * @param {Array} `array`
+ * @param {any} `value`
+ * @param {Object} `options`
+ * @return {String}
+ * @block
+ * @api public
+ */
+
+helpers.groupEach = function (every, context, options) {
```

Comment:
The arguments should be `(context, every, options)` to match the other array helpers.
Also, change `context` to `array` to match the other helpers.
The code comments above need to be updated with the correct name and parameter information.